src/bin/qxmoji/main.c

```diff
@@ -1,3 +1,8 @@
+/* Needed to build on Linux */
+#ifdef __linux__
+#	define _GNU_SOURCE
+#endif
```

Comment:
See my previous comment, `_GNU_SOURCE` should better be avoided for portable code (and guarding it in an `#ifdef` is a workaround and not really correct either, as there are Linux systems using other libc implementations than `glibc`)...

Nevertheless this is a real issue that should be fixed for the first release, I suggest `#define _POSIX_C_SOURCE 200112L` instead, probably also in `singleinstance.cpp`.